packages/counterfactual_nft/contracts/CounterfactualNFT.sol

```diff
@@ -69,29 +75,30 @@ contract CounterfactualNFT is ICounterfactualNFT, Initializable, ERC1155Upgradea
         // in `minters` and `isMinter`.
     }
 
-    function mint(
-        address       account,
-        uint256       id,
-        uint256       amount,
-        bytes  memory data
-        )
-        external
-        onlyFromMinter
-    {
-        _mint(account, id, amount, data);
-    }
+    /// update at 2022-05-31: Do not allow minting on Layer 1
```

Comment:
Any particular reason to disable minting on L1?

Reply:
> Any particular reason to disable minting on L1?

This functionality is never used and is not expected to be used in the future.

Reply:
Is there any downside of keeping it?

Reply:
> 

The backend will make sure a tokenId can be minted only once, this method can break it.